JetUtilities/src/crystalBall.cc

```diff
@@ -0,0 +1,227 @@
+#include "JetMETAnalysis/JetUtilities/interface/crystalBall.h"
+
+#include "TMath.h"
+
```

Comment:
I prefer keep it where I'm using it to do not go back and forth to check 
o set the header.
More over this is my code, so I prefer keep my style.

Le 27/04/2016 03:24, Alexx Perloff a écrit :
>
> In JetUtilities/src/crystalBall.cc 
> <https://github.com/cms-jet/JetMETAnalysis/pull/10#discussion_r61187055>:
>
> > @@ -0,0 +1,227 @@
> > +#include "JetMETAnalysis/JetUtilities/interface/crystalBall.h"
> > +
> > +#include "TMath.h"
> > +
>
> As a style I like to keep all of the headers in the header file of the 
> class that uses them. It also prevents multiple linking issues.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-jet/JetMETAnalysis/pull/10/files/abbd2ce4a9b0377684ddf51b4382b801e0a66c05#r61187055>
>